packages/salesforcedx-utils-vscode/src/cli/configUtil.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
```

Comment:
```suggestion
 * Copyright (c) 2020, salesforce.com, inc.
```

Reply:
Same "year" change needed in other files as well.